<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QWidget" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QLineEdit" name="leMetal"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lePrecio"/>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cbInstruments"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leMarca"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leCCuerdas"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Marca</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Cantidad de Teclas</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Cantidad de Cuerdas</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Precio</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Metal Usado</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leCTeclas"/>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ver Stock</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>AÃ±adir</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
